protocol/communities/manager.go

```diff
@@ -2053,8 +2079,29 @@ func (m *Manager) ExtractMessagesFromHistoryArchives(communityID types.HexBytes,
 
 		err := proto.Unmarshal(data, archive)
```

Comment:
Are you sure this fails for encrypted archives?
If they are both protobuf structs encoded, but different types, generally it doesn't fail, but will be an empty struct with some extra fields

Reply:
Yea so this is interesting.
I've noticed that, when it's decrypted on the community owner side, it indeed doesn't error out. But on the community member side it will error.

Will verify this once more, but this is why I ended up handling it this way.